support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -127,17 +152,18 @@ function updateAmount(amount: number, paymentRequest: Object | null) {
 function onClick(event, props: PropTypes) {
   event.preventDefault();
   trackComponentClick('apple-pay-clicked');
-  updateAmount(props.amount, props.stripePaymentRequestObject);
-  props.setStripePaymentRequestButtonClicked();
+  updateTotal(props);
+  props.setAssociatedPaymentMethod();
```

Comment:
why was this call not necessary before?

Reply:
Because there is a special thank you page for direct debit.